app/src/main/java/com/github/steroidteam/todolist/todo/Task.java

```diff
@@ -0,0 +1,64 @@
+package com.github.steroidteam.todolist.todo;
+
+import java.io.Serializable;
+
+/**
+ * A class that represents a task described by a title and a body.
+ */
+public class Task implements Serializable{
+
+    private String body;
+    private final String id;
+
+    /**
+     * Constructs a new Task with a title and a body.
+     *
+     * @param id The id of the task.
+     * @param body The body of the task.
+     * @throws IllegalArgumentException Thrown if one or more arguments are null
+     */
+    public Task(String id, String body) {
+        if(id == null || body == null) {
+            throw new IllegalArgumentException();
+        }
+
+        this.id = id;
+        this.body = body;
+    }
+
+    /**
+     * Modifies the body of the task.
+     * @param newBody The new body.
+     */
+    public void modifyBody(String newBody) {
```

Comment:
Following Java's convention of getters and setters, I feel like `setBody` is a better name here. What do you think?

Reply:
For sure, I will follow getters and setters convention.